src/wranglerjs/mod.rs

```diff
@@ -183,6 +183,23 @@ pub fn run_npm_install(dir: PathBuf) -> Result<(), failure::Error> {
     }
 }
 
+// build a Command for npm
+//
+// Here's the deal: on Windows, `npm` isn't a binary, it's a shell script.
+// This means that we can't invoke it via `Command` directly on Windows,
+// we need to invoke `cmd /C npm`, to run it within the cmd environment.
+fn build_npm_command() -> Command {
+    if install::target::WINDOWS {
```

Comment:
why choose `install::target::WINDOWS` over `cfg!(target_os = "windows")`?

Reply:
this would be the one thing i'd change but we can do it in another PR i think

Reply:
I used it for consistency, but we can change it.